packages/reporter/src/hooks/hook-model.spec.js

```diff
@@ -109,23 +109,42 @@ describe('Hook model', () => {
   })
 
   context('#aliasesWithDuplicates', () => {
+    const addCommand = (alias, hasDuplicates = false) => {
+      return hook.addCommand({
+        isMatchingEvent: () => {
+          return false
+        },
+        alias,
+        hasDuplicates,
+      })
+    }
+
     it('returns duplicates marked with hasDuplicates and those that appear mulitple times in the commands array', () => {
-      hook.addCommand({ isMatchingEvent: () => {
-        return false
-      }, alias: 'foo' })
-      hook.addCommand({ isMatchingEvent: () => {
-        return false
-      }, alias: 'bar' })
-      hook.addCommand({ isMatchingEvent: () => {
-        return false
-      }, alias: 'foo' })
-      hook.addCommand({ isMatchingEvent: () => {
-        return false
-      }, alias: 'baz', hasDuplicates: true })
+      addCommand('foo')
+      addCommand('bar')
+      addCommand('foo')
+      addCommand('baz', true)
 
       expect(hook.aliasesWithDuplicates).to.include('foo')
       expect(hook.aliasesWithDuplicates).to.include('baz')
       expect(hook.aliasesWithDuplicates).to.not.include('bar')
     })
+
+    it('returns the same array instance if it has not changed', () => {
```

Comment:
can you leave a comment pointing to the github issue that this fixes? it would help correlate the test - as the meaning as to "why" it needs to return the same array is completely lost without that context...

`https://github.com/cypress-io/cypress/issues/4411`